avldb/src/main/scala/scorex/db/LDBFactory.scala

```diff
@@ -1,92 +1,81 @@
 package scorex.db
 
-import java.io.File
-import java.util.concurrent.locks.ReentrantReadWriteLock
-import org.iq80.leveldb.{DB, DBFactory, DBIterator, Options, Range, ReadOptions, Snapshot, WriteBatch, WriteOptions}
+import org.rocksdb._
+import org.rocksdb.util.SizeUnit
 import scorex.util.ScorexLogging
 
+import java.io.File
+import java.util.concurrent.atomic.AtomicBoolean
+import java.util.concurrent.locks.ReentrantReadWriteLock
 import scala.collection.mutable
 
 /**
-  * Registry of opened LevelDB instances.
-  * LevelDB prohibit access to the same storage file from more than one DB instance.
+  * Registry of opened RocksDB instances.
+  * RocksDB prohibit access to the same storage file from more than one DB instance.
   * And ergo application (mostly tests) quite frequently doesn't not explicitly close
   * database and tries to reopen it.
   */
-case class StoreRegistry(factory: DBFactory) extends DBFactory with ScorexLogging {
-
-  val lock = new ReentrantReadWriteLock()
-  val map = new mutable.HashMap[File, RegisteredDB]
-
-  /**
-    * Decorator of LevelDB DB class which overrides close() methods and unlinks database from registry on close.
-    * So if database was not explicitly closed, then next attempt to open database with the same path will
-    * return existed instance instead of creating new one.
-    */
-  case class RegisteredDB(impl: DB, path: File) extends DB {
-
-    def get(key: Array[Byte]): Array[Byte] = impl.get(key)
-
-    def get(key: Array[Byte], options: ReadOptions): Array[Byte] = impl.get(key, options)
+object LDBFactory extends ScorexLogging {
```

Comment:
LDBFactory => RocksDBFactory (as L means Level)

Reply:
changed all occurrences of LDB to RocksDB